next-app/src/lib/progress/index.ts

```diff
@@ -2,7 +2,7 @@ import { writable } from 'svelte/store'
 
 export const loading = writable(false)
 
-const pending = []
+const pending: any[] = []
```

Comment:
I don't see the point of `any`... I'd more inclined to force a `string` if you feel something is needed here.

Reply:
Sure, it could be `string[]`. I used `any[]` because I didn't know what types would be used here and this was the quickest way to get Typescript to stop complaining. But `string[]` probably does make more sense.